frontend/src/components/shared/AlertContainer/AlertContainer.tsx

```diff
@@ -27,7 +27,13 @@ export enum Kind {
   WARNING = "warning",
 }
 
-function getNotificationKind(kind: Kind): KIND[keyof KIND] {
+function getNotificationKind(
+  kind: Kind
+):
+  | typeof KIND.negative
```

Comment:
Do you know why the previous return type did not work anymore?

Reply:
They've changed `KIND` type, see:

![image](https://user-images.githubusercontent.com/78742618/187902128-69cf7eb9-d903-4429-8265-21a4580e6733.png)

after suggested change:

![image](https://user-images.githubusercontent.com/78742618/187902298-bcda0600-2cd7-490f-b6f8-f592b28da005.png)